.github/workflows/test.yaml

```diff
@@ -19,12 +19,15 @@ jobs:
 
   environments:
     runs-on: ubuntu-latest
-    name: ${{ matrix.env }} - v${{ matrix.version }}
+    name: ${{ matrix.env }}${{ matrix.env == 'node' && '@' || ''}}${{ matrix.env == 'node' && matrix.version || ''}}
     strategy:
       fail-fast: false
       matrix:
-        env: [node]
+        env: [node, browser]
         version: [18.18.2, 20]
+        exclude:
+          - env: browser
+            version: 18.18.2
```

Comment:
Tricky.

Reply:
Could have gone for for a matrix that uses an array of objects if we prefer and conditionally check for the presence of a versions key. However [the docs](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#excluding-matrix-configurations) seems to prefer `include`/`include` keyword usage rather than using conditionals. I am indifferent.

Reply:
How would it look with an array of objects?

Reply:
[Like this](https://github.com/FuelLabs/fuels-ts/pull/1630/commits/d918fadc857c4156e49746c4c75c93e98581eb1f)